src/stylesheet.rs

```diff
@@ -233,7 +235,12 @@ where
     let project_root = options.project_root.clone();
     let mut printer = Printer::new(&mut dest, options);
 
-    printer.sources = Some(&self.sources);
+    #[cfg(feature = "sourcemap")]
+    {
```

Comment:
Conditionally compiling expressions isn't stable, so needed to wrap this in a block.